product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,265 @@
+
+```
+echo -e "track_commit_timestamp = 'on'" | sudo -u enterprisedb tee -a /var/lib/edb/as16/data/postgresql.conf >/dev/null
+```
+
+#### Add a password to the Postgres enterprisedb user
+
+To allow connections between nodes, a password needs to be set on the Postgres enterprisedb user. 
+For this example, we are using the password `secret`.
+Select a different password for your deployments.
+You will need this password when we get to [Connecting the Cluster](05-connecting-cluster).
+
+```
+sudo -u enterprisedb psql edb -c "ALTER USER enterprisedb WITH PASSWORD 'secret'"
+```
+
+#### Raise `max_worker_processes`
+
+To communicate between multiple nodes, Postgres Distributed nodes run more worker processes than usual.
+The default limit (8) is too low even for a small cluster.
+
+Raise the maximum number of worker processes with this commmand:
+
+```
+echo -e "max_worker_processes = '16'" | sudo -u enterprisedb tee -a /var/lib/edb/as16/data/postgresql.conf >/dev/null
+```
+
+The `max_worker_processes` value should be raised for larger clusters.
+To calculate the needed value see [Postgres configuration/settings](../postgres-configuration/#postgres-settings).
+
+#### Enable node communications in pg_hba.conf
+
+Out of the box, Postgres allows local communications with the database but not external network connections. 
+To enable this, edit `pg_hba.conf` and add appropriate rules, including rules for the replication users.
+To simplify the process, use this command:
+
+```
+echo -e "host all all all md5\nhost replication all all md5" | sudo tee -a /var/lib/edb/as16/data/pg_hba.conf
+```
+
+It will append
+
+```
+host all all all md5
+host replication all all md5
+```
+
+to `pg_hba.conf` which will enable the nodes to replicate.
+
+#### Enable authentication between nodes
+
+As part of the process of connecting nodes for replication, PGD logs into other nodes. 
+It will perform that log in as the user that Postgres is running under.
+For epas, this is the `enterprisedb` user.
+That user will need credentials to log into the other nodes.
+We will supply these credentials using the `.pgpass` file which needs to reside in the user's home directory.
+The home directory for `enterprisedb` is `/var/lib/edb`.
+
+Run this command to create the file:
+
+```
+echo -e "*:*:*:enterprisedb:secret" | sudo -u enterprisedb tee /var/lib/edb/.pgpass; sudo chmod 0600 /var/lib/edb/.pgpass
+```
+
+### Restart the server
+
+After all these configuration changes, it is recommended that the server is restarted with:
+
+```
+sudo systemctl restart edb-as-16
+```
+
+### Create the replicated database
+
+The server is now prepared for PGD. 
+We need to next create a database named `bdrdb` and install the bdr extension when logged into it.
+
+```
+sudo -u enterprisedb psql edb -c "CREATE DATABASE bdrdb"
+sudo -u enterprisedb psql bdrdb -c "CREATE EXTENSION bdr"
+```
+
+Finally, test the connection by logging into the server.
```

Comment:
Given that the preceding command would only work if the connection is functional, IMO it's not really necessary to do that again. What would be useful is to get a quick idea of what to expect when logging in, e.g.:

Finally, log into the server and check the extension was installed with the `\dx` command:
```
$ sudo -u enterprisedb psql bdrdb
psql (16.0.0)
Type "help" for help.

postgres=# \dx
                                 List of installed extensions
       Name       |  Version  |   Schema   |                   Description
------------------+-----------+------------+--------------------------------------------------
 bdr              | 6.0.0-dev | pg_catalog | Bi-Directional Replication for PostgreSQL
 edb_dblink_libpq | 1.0       | pg_catalog | EnterpriseDB Foreign Data Wrapper for PostgreSQL
 edb_dblink_oci   | 1.0       | pg_catalog | EnterpriseDB Foreign Data Wrapper for Oracle
 edbspl           | 1.0       | pg_catalog | EDB-SPL procedural language
 plpgsql          | 1.0       | pg_catalog | PL/pgSQL procedural language
(5 rows)
```

Reply:
Like that but less... 6.0.0-dev-y. Done.